internal/rpcserver/rpcserver.go

```diff
@@ -6291,8 +6312,8 @@ func (s *Server) route(ctx context.Context) *http.Server {
 		// Keep track of the number of connected clients.
 		s.incrementClients()
 		defer s.decrementClients()
-		_, isAdmin, err := s.checkAuth(r, true)
-		if err != nil {
+		authed, isAdmin := s.checkAuth(r, true)
```

Comment:
Notice how this non-Websocket endpoint requires authentication via HTTP basic auth.

Reply:
Reverted.